website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-slack-notifications.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Configuring Slack Notifications"
-description: "You can report status alerts in a designated Slack channel every time you run a job."
+title: "Configuring Slack notifications"
+description: "Report status alerts in a designated Slack channel every time you run a job."
```

Comment:
Why is there notifications and Slack notifications as two different pages? I haven't looked at their content, just noticed during the review